Bio/SeqFeature.py

```diff
@@ -64,6 +64,7 @@
 import functools
 import re
 import warnings
+import copy
```

Comment:
Do you still need this import?

Reply:
Oops, no that's just a remnant from me not being careful cleaning up